src/main/java/hudson/plugins/xvnc/DisplayAllocator.java

```diff
@@ -18,6 +14,9 @@
  * @author Kohsuke Kawaguchi
  */
 final class DisplayAllocator {
+
+    transient Saveable owner;
```

Comment:
IIUC we use this solely to notify the owner of the `save` events. I suggest to put it in through constructor and encapsulate properly. I not not mind if it stays nullable to for sustain BC.